redwood-treehouse-composeui/src/commonMain/kotlin/app/cash/redwood/treehouse/composeui/TreehouseContent.kt

```diff
@@ -52,6 +52,8 @@ public fun <A : AppService> TreehouseContent(
       override val widgetSystem = widgetSystem
       override val readyForContent = true
       override var readyForContentChangeListener: ReadyForContentChangeListener? = null
+      override var saveCallback: TreehouseView.SaveCallback? = null
+      override val restoredId: String? = null
```

Comment:
For follow-up, lets do a value class for these. 

Example from redwood-protocol:
https://github.com/cashapp/redwood/blob/afe1c9f5f95eec3cff46837a4b2749cbaf72af8b/redwood-protocol/src/commonMain/kotlin/app/cash/redwood/protocol/values.kt#L32

Reply:
Maybe call it `StateSnapshot.Id` ??